corehq/apps/locations/templates/locations/manage/import.html

```diff
@@ -1,10 +1,10 @@
-{% extends "settings/bootstrap2/base_template.html" %}
+{% extends "settings/bootstrap3/base_template.html" %}
 {% load hq_shared_tags %}
 
 {% block js %} {{ block.super }}
-    <script src="{% static 'commtrack/ko/location_bulk_upload_file.js' %}"></script>
+    <script src="{% new_static 'commtrack/ko/location_bulk_upload_file.js' %}"></script>
 {% endblock %}
 
-{% block main_column %}
-    {% include "locations/manage/partials/bulk_upload.html" %}
+{% block page_content %}
+    {% include "style/bootstrap3/partials/bulk_upload.html" %}
```

Comment:
The original template `"locations/manage/partials/bulk_upload.html"` includes a couple extra options for the download which are removed here.  That page extendes `"style/bootstrap2/partials/bulk_upload.html"` - I think you should be able to just switch it over to bootstrap 3 there, but we do need those checkboxes.